lib/src/directus_api_manager_base.dart

```diff
@@ -29,6 +29,26 @@ class DirectusApiManager implements IDirectusApiManager {
   @override
   String? get accessToken => _api.accessToken;
 
+  @override
+  String? get refreshToken => _api.refreshToken;
+  set refreshToken(String? value) => _api.refreshToken = value;
+
+  @override
+  String get webSocketBaseUrl {
+    // Remove last / if present
+    String url = _api.baseUrl;
+    if (url.endsWith("/")) {
+      return url.substring(0, _api.baseUrl.length - 1);
+    }
+    if (url.startsWith("http")) {
```

Comment:
this will always match the first if because https does start with http.
This should work though, but you can remove the else trying to find an https start as it's dead code.

Reply:
done in bfa2c9cc3a8c5c4f91a2a0ec953631c42602edcf